### PR TITLE
feat: Interface implementation(multimedia info and convert)

### PR DESCRIPTION
**Pull request by myk1343**
Interface implementation(multimedia info and convert)

Log: Interface implementation(multimedia info and convert)

### CODE REVIEW

File: examples/mideaConvertAndInfo/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# SPDX-FileCopyrightText: 2022 UnionTech Software Technology Co., Ltd.
```
Comment (xzl01) [inline examples/mideaConvertAndInfo/CMakeLists.txt]: 2023

File: examples/mideaConvertAndInfo/main.cpp [hunk 2]
```diff
@@ -0,0 +1,14 @@
+// SPDX-FileCopyrightText: 2022 UnionTech Software Technology Co., Ltd.
```
Comment (xzl01) [inline examples/mideaConvertAndInfo/main.cpp]: 同上

File: examples/mideaConvertAndInfo/CMakeLists.txt (hunk 1 above)
Comment (myk1343) [reply examples/mideaConvertAndInfo/CMakeLists.txt]: 已修改

File: examples/mideaConvertAndInfo/main.cpp (hunk 2 above)
Comment (myk1343) [reply examples/mideaConvertAndInfo/main.cpp]: 已修改

File: include/dtkmultimedia/dmultioperateinterface.h [hunk 3]
```diff
@@ -1,4 +1,4 @@
-// SPDX-FileCopyrightText: 2022 UnionTech Software Technology Co., Ltd.
+// SPDX-FileCopyrightText: 2023 UnionTech Software Technology Co., Ltd.
```
Comment (xzl01) [inline include/dtkmultimedia/dmultioperateinterface.h:1]: 其实严格来说 如果是对现有文件进行修改 应该标注为2022 - 2023

**Review by xzl01 — COMMENTED** [review]
+1

**Comment by toberyan** [comment]
+1

**Review by Groveer — COMMENTED** [review]
+1

**Review by xzl01 — APPROVED** [review]

**Review by Groveer — APPROVED** [review]

**Pull request merged**